CHANGELOG.md

```diff
@@ -82,109 +90,147 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
-    ([\#1639](https://github.com/seqan/seqan3/pull/1639)).
-  * The configuration element `seqan3::search_cfg::output` does not exist anymore. It has been replaced by the
-    individual configuration elements
-    * `seqan3::search_cfg::output_query_id`
-    * `seqan3::search_cfg::output_reference_id`
-    * `seqan3::search_cfg::output_reference_begin_position`
-    * `seqan3::search_cfg::output_index_cursor`
-    see the \ref search_configuration_subsection_output "output configuration" for further details.
+  * The configuration `seqan3::search_cfg::max_error` has been replaced by individual configuration elements:
+    * `seqan3::search_cfg::max_error{seqan3::search_cfg::total}` to `seqan3::search_cfg::max_error_total{}`
+    * `seqan3::search_cfg::max_error{seqan3::search_cfg::insertion}` to `seqan3::search_cfg::max_error_insertion{}`
+    * `seqan3::search_cfg::max_error{seqan3::search_cfg::deletion}` to `seqan3::search_cfg::max_error_deletion{}`
+    * `seqan3::search_cfg::max_error{seqan3::search_cfg::substitution}` to
+      `seqan3::search_cfg::max_error_substitution{}` ([\#1861](https://github.com/seqan/seqan3/pull/1861)).
+  * The max error configurations can be initialised with either a `seqan3::search_cfg::error_rate` or
+    `seqan3::search_cfg::error_count`, and can be reassigned ([\#1861](https://github.com/seqan/seqan3/pull/1861)).
+  * The configuration `seqan3::search_cfg::mode` has been replaced by individual configuration elements
+    ([\#1639](https://github.com/seqan/seqan3/pull/1639)):
+    * `seqan3::search_cfg::mode{seqan3::search_cfg::all}` to `seqan3::search_cfg::hit_all{}`
+    * `seqan3::search_cfg::mode{seqan3::search_cfg::best}` to `seqan3::search_cfg::hit_single_best{}`
+    * `seqan3::search_cfg::mode{seqan3::search_cfg::all_best}` to `seqan3::search_cfg::hit_all_best{}`
+    * `seqan3::search_cfg::mode{seqan3::search_cfg::strata{5}}` to `seqan3::search_cfg::hit_strata{5}`
+    * The `seqan3::search_cfg::hit_strata` member variable `value` has been replaced to `stratum`
+  * The configuration`seqan3::search_cfg::output` has been replaced by individual configuration elements
+    ([\#1862](https://github.com/seqan/seqan3/pull/1862)):
+    * `seqan3::search_cfg::output{seqan3::search_cfg::text_position}` to `seqan3::search_cfg::output_reference_begin_position{}`
+    * `seqan3::search_cfg::output{seqan3::search_cfg::text_position}` to `seqan3::search_cfg::output_index_cursor`
+    * `seqan3::search_cfg::output_query_id{}` has been added
+    * `seqan3::search_cfg::output_reference_id{}` has been added
 * Removed `seqan3::bi_fm_index_cursor::to_rev_cursor()` and `seqan3::bi_fm_index::rev_cursor()`
   ([\#1892](https://github.com/seqan/seqan3/pull/1892)).
 
+Header Changes:
+
+```cpp
+#include <seqan3/search/{algorithm => }/search.hpp>
+#include <seqan3/{search/search_result_range.hpp => core/algorithm/algorithm_result_generator_range.hpp}>
+
+#include <seqan3/search/configuration/{mode.hpp => hit.hpp}>
```

Comment:
```suggestion
#include <seqan3/{search/search_result_range.hpp => core/algorithm/algorithm_result_generator_range.hpp}>
#include <seqan3/search/configuration/{mode.hpp => hit.hpp}>
```
?

Reply:
There's always a new line if the subdirectory changes (`search/` and `search/configuration`)

Reply:
ah ok